Xamarin.Essentials/Types/PlatformExtensions/ColorExtensions.mac.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Drawing;
+using AppKit;
+
+namespace Xamarin.Essentials
+{
+    public static partial class ColorExtensions
+    {
+        public static Color ToSystemColor(this NSColor color)
+        {
+            color.GetRgba(out var red, out var green, out var blue, out var alpha);
```

Comment:
uncommon but you can do `(null as NSColor).ToSystemColor ()` and it would throw an NRE

Reply:
Thanks.